### PR TITLE
News App: GET and POST Request

### PR DESCRIPTION
**Pull request by LehabuL**
The app has both GET and POST requests and a user interface. The code for the app needs to be reviewed by 2 people before merging as discussed in [Meeting 15](https://github.com/bounswe/bounswe2022group6/wiki/Meeting-%2315-05.06.2022).

### CODE REVIEW

**Comment by LehabuL** [comment]
There were some minor conflicts with the master branch so I just resolved them

File: practice_app/practice_app/settings.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     'rest_framework',
     'apps.advice',
     'apps.article',
+    'apps.news'
```
Comment (Aliqaan) [inline practice_app/practice_app/settings.py:44]: Missing comma in the end causing errors.

File: practice_app/practice_app/urls.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
     path('admin/', admin.site.urls),
     path('advice/', include('apps.advice.urls')),
     path('article/', include('apps.article.urls')),
+    path('news/', include('apps.news.urls'))
```
Comment (Aliqaan) [inline practice_app/practice_app/urls.py:23]: There is a missing comma aswell.

**Review by Aliqaan — CHANGES_REQUESTED** [review]
Apart from the missing commas I mentioned, your app is looking great and works fine.

**Comment by haticerk** [comment]
I have some suggestions to improve the user experience. 

* When entering a country,  the system does not require me to choose from the list. And when I type a country that not matches the list, it crashes. You may force user to select from the list.

* When there is no news found with given keyword, it just returns a blank page. Maybe, You can add a warning tells that no related news found with the keyword.

Apart from these, app works correctly and your user interface looks cool, great job!

File: practice_app/practice_app/settings.py (hunk 1 above)
Comment (LehabuL) [reply practice_app/practice_app/settings.py:44]: it was due to the inconsistency between master, sorry I did not double check the commas.

**Review by Aliqaan — APPROVED** [review]

**Comment by LehabuL** [comment]
I corrected the issues that you mentioned, @haticerk can you reinspect?

**Review by haticerk — APPROVED** [review]
Thank you for your effort. Great job!

**Pull request merged**